config/prow/external-secrets/secretstore_update.yaml

```diff
@@ -9,5 +9,5 @@ spec:
       auth:
         secretRef:
           secretApiKeySecretRef:
-            name: ibm-secret
-            key: apiKey
+            name: ibm-cloud-credentials
```

Comment:
if you use `namespace` as well here in this spec, then you don't need to create copy of this secret everywhere!

of course you might need to add some additional role to read that secret from a different namespace!

Reply:
I shall see how to do this.

Reply:
@mkumatag I get the below error when I use `namespace` in this spec. (Tried to use secret from `test-pods` namespace to `perfdash` namespace)
```
[root@toad1 test-external-secrets]# kubectl apply -f secretstore_update.yaml -n perfdash
Error from server (namespace not allowed with namespaced SecretStore): error when creating "secretstore_update.yaml": admission webhook "validate.secretstore.external-secrets.io" denied the request: namespace not allowed with namespaced SecretStore
[root@toad1 test-external-secrets]#
```
Also, when you say "_**you might need to add some additional role to read that secret from a different namespace**_" , is it to the APIKey through which we log into IBMCloud and apply the above spec?

Can you please elaborate on how to do it...

Reply:
@mkumatag Can you please take a look.

Reply:
Have changed to `ClusterSecretStore`